docs/content/guides/installation/upgrade/index.md

```diff
@@ -0,0 +1,174 @@
+---
+type: docs
+title: "How-To: Upgrade or Roll Back Radius on Kubernetes"
+linkTitle: "Upgrade & Rollback"
+description: "Upgrade the Radius control plane and revert to a previous revision when required"
+weight: 300
+categories: "How-To"
+tags: ["Kubernetes"]
+aliases:
+  - /guides/operations/kubernetes/kubernetes-upgrade
+  - /guides/operations/kubernetes/kubernetes-rollback
+---
+
+Radius upgrades drives a Helm upgrade of the Radius control plane (and Contour, if installed), while rollbacks replays an earlier Helm revision. Both these operations target the `radius` Helm release in the `radius-system` namespace.
+
+## Upgrade Radius
+
+### Before you run `rad upgrade`
+
+- Install the latest `rad` CLI. The CLI release determines the default chart version pushed to the cluster.
```

Comment:
This is not correct. You can use a rad CLI that has `rad upgrade/rollback kubernetes` commands and use that to upgrade/rollback to any version.